src/treeview/tree-node.types.ts

```diff
@@ -7,6 +7,7 @@ export interface Node {
 	id?: string;
 	active?: boolean;
 	disabled?: boolean;
+	selectable?: boolean;
```

Comment:
Could you add a comment to what `selectable` is here? This is so devs don't have to trace the code to figure out what this does.